src/main/kotlin/no/nav/familie/ef/iverksett/lagreIverksett/infrastruktur/LagreIverksettJdbc.kt

```diff
@@ -0,0 +1,79 @@
+package no.nav.familie.ef.iverksett.lagreIverksett.infrastruktur
+
+import arrow.core.Either
+import no.nav.familie.ef.iverksett.domene.Brev
+import no.nav.familie.ef.iverksett.lagreIverksett.tjeneste.KunneIkkeLagreIverksett
+import no.nav.familie.ef.iverksett.lagreIverksett.tjeneste.LagreIverksett
+import org.slf4j.LoggerFactory
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate
+import org.springframework.jdbc.support.GeneratedKeyHolder
+import org.springframework.jdbc.support.KeyHolder
+import java.sql.Connection
+import java.sql.SQLException
+import java.util.*
+
+class LagreIverksettJdbc(val namedParameterJdbcTemplate: NamedParameterJdbcTemplate) : LagreIverksett {
+
+    private val logger = LoggerFactory.getLogger(javaClass)
+
+    companion object {
+        val IVERKSETT_JSON_VERSJON: Int = 1
+    }
+
+    override fun lagre(iverksettJson: String, brev: List<Brev>): Either<KunneIkkeLagreIverksett, Int> {
+        try {
+            return Either.Right(lagreIverksett(iverksettJson, brev))
+        } catch (exception: Exception) {
+            logger.error("Kunne ikke lagre iverksett json : ${exception}")
```

Comment:
Vet vi hva exception inneholder? Burde vi logge til securelog her?

Reply:
Ja, kanskje man burde gjøre det, hvis noe av dataen faller igjennom på et vis ? Satser på å gjøre det, for sikkerhets skyld (så kan vi endre tilbake evt)